backend/helper_functions.py

```diff
@@ -328,6 +328,175 @@ def user_validation(data):
     return None, status.HTTP_200_OK
 
 
+# Validate the recipe verifying it has the correct fields
+def recipe_validation(data):
```

Comment:
This function is good for the meantime while we are still implementing routes, but it will soon be unnecessary as the front end will validate the requests.